specs/market-api.yaml

```diff
@@ -106,58 +98,45 @@ paths:
         This implies, that client code should not `unsubscribeOffer` before it has received
         all expected/useful inputs from `collectDemands`.
```

Comment:
`collectDemands` in Offers endpoint.

Reply:
It is correct. Because `Offer` issuer (Provider) is waiting for matching `Demand`s.